config/locales/gu/organisations.yml

```diff
@@ -331,8 +331,6 @@ gu:
       special_health_authority:
       sub_organisation:
       tribunal: ટ્રિબ્યુનલ
-    view_all: બધા જુઓ
```

Comment:
@maxgds / @jon-kirwan I'm thinking - this content translation work has already been done for this phrase. Might it make sense to note to update the details component with this?  Or even just capture this for when localisation is prioritised?

Reply:
I'm not sure I follow, Chris? At a glance the details component looks like it requires a string input and doesn't have a default. Am I missing something?

Reply:
🤔 Maybe I am, let me explain in more detail -  so the details element here on `collections` before this change is bespoke, however the phrase "view all" has already had Content translation work done to pull in the different text should the language be changed

As we're swapping to the gem version what happens with all these translations. How does a user see `બધા જુઓ`?

Reply:
So any app that has copy in it should have had the copy put into a translation file and the translations should have been sourced and updated for all relevant languages. I'd be surprised if the phrase "view all" doesn't appear anywhere else across GOV.UK but it's possible. If that's the case, yes, we're throwing away some translation work with this deletion but that's better than leaving it to clutter up the yaml files when it's not needed. I guess we could capture this deletion by linking to this PR in a doc somewhere in case it ever needs to be reimplemented, but I'm not sure we'd be able to communicate that to everyone who might ever remove a translation key and we don't know for sure that we'll ever need it again. And when we look to add "view all" in future we *could* look for it in a linked doc, but it's likely people will forget to... I think we'd just be adding overhead if we were to try to capture this so favour just letting it go, to be honest.

Reply:
Fair - also it's in git so if someone does go looking (they might not find it) but it's there.

Reply:
Thanks, i'll make sure to capture this in a separate commit too which might it a bit easier to find if needed.